file/fs.lisp

```diff
@@ -71,18 +82,36 @@
                (error 'unknown-host
                       :host name
                       :pathname (format nil "~A:" name))))))
-    (t name)))
-
-(defun (setf find-host) (new-value name &optional errorp)
-  (declare (ignore errorp))
+    (file-system-host name)
+    (t (when errorp
```

Comment:
This should be an unconditional error. Errorp determines if an error is signalled if the named host does not exist (like find-class) and should not have an effect on misuses of the function.